Cakefile

```diff
@@ -75,6 +75,7 @@ task "package", "Builds a zip file for submission to the Chrome store. The outpu
     blacklist.map((item) -> ["--exclude", "#{item}"]))
 
   spawn "rsync", rsyncOptions, false, true
+  spawn "sed", "-i /clipboardWrite/d dist/vimium/manifest.json".split /\s+/
```

Comment:
Won't doing this here (before packaging for either FF or Chrome) mean removing the permission for both..?